webui/react/src/components/PageHeader.tsx

```diff
@@ -17,12 +17,49 @@ export interface Props extends CommonProps {
   title?: string;
 }
 
+const tooLongExperimentDescription = 40;
+
+const isExperimentIdWithDescCrumb = (breadcrumbName: string) => {
+  const crumbElements = breadcrumbName.split(' ');
+  // The experiment description may have spaces in it, so there should be at least 3 elements
+  if (crumbElements.length < 3
+      || crumbElements[0] !== 'Experiment'
```

Comment:
To make breadcrumb Tooltips something re-usable we could:
- extend the `BreadCrumbRoute` and add a `breadcrumbTooltip: string` property
- here, if there's a `breadcrumbTooltip`, render the `<Tooltip>` component
- in `TrialDetails.tsx` pass this additional prop for the exp breadcrumb item and keep the related logic there (add max 40 chars of description to `breadcrumbName`)

Reply:
That's a much better implementation. Updated the PR to use that approach